firecracker-control/vsock.go

```diff
@@ -56,7 +56,7 @@ func findNextAvailableVsockCID(ctx context.Context) (*os.File, uint32, error) {
 			return nil, 0, ctx.Err()
 		default:
 			cid := contextID
-			_, _, err = syscall.Syscall(
+			_, _, err = sysCall(
```

Comment:
Why was this changed?

Reply:
Because the linter was complaining that sysCall wasn't being use anywhere. Then I realized our tests were using it, except the tests would use the actual `syscall.Syscall` due to the line above.

Reply:
Since this is unrelated to the rest of your PR, can you revert it here and address it separately?